scss/_choose-us.scss

```
@@ -0,0 +1,77 @@
+.choose-us {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
```

```suggestion
  min-height: 100vh;
```
чтоб не уменьшалось больше, чем нужно